examples/04_glm_first_level/plot_two_runs_model.py

```diff
@@ -0,0 +1,271 @@
+#
+# We can use :func:`~nilearn.glm.compute_fixed_effects` to compute
+# the fixed effects statistics using the outputs
+# from the run-specific FirstLevelModel results.
+from nilearn.glm.contrasts import compute_fixed_effects
+
+contrast_imgs = [
+    summary_statistics_ses1["effect_size"],
+    summary_statistics_ses2["effect_size"],
+]
+variance_imgs = [
+    summary_statistics_ses1["effect_variance"],
+    summary_statistics_ses2["effect_variance"],
+]
+
+fixed_fx_contrast, fixed_fx_variance, fixed_fx_stat = compute_fixed_effects(
+    contrast_imgs,
+    variance_imgs,
+    data["mask"],
+)
+plotting.plot_stat_map(
+    fixed_fx_stat,
+    bg_img=mean_img_,
+    threshold=3.0,
+    cut_coords=cut_coords,
+    title=f"{contrast_id}, fixed effects",
+)
+
+# %%
+# Not unexpectedly, the fixed effects version displays higher peaks
+# than the input runs.
+# Computing fixed effects enhances the signal-to-noise ratio
+# of the resulting brain maps.
+
+# %%
+# Compute the fixed effects statistics
+# using the preprocessed data of both runs.
+#
+# A more straightforward alternative to fitting run-specific GLMs,
+# than combining the results with :func:`~nilearn.glm.compute_fixed_effects`,
+# is to simply fit the GLM to both runs at once.
+#
+# Since we can assume that the design matrices of both runs
+# have the same columns, in the same order,
+# we can again reuse the first run's contrast vector.
+fmri_glm_multises = fmri_glm.fit(fmri_imgs, design_matrices=design_matrices)
+
+# We can just define the contrast array for one run and assume
+# that the design matrix is the same for the other.
+# However, if we want to be safe, we should define each contrast separately,
+# and provide it as a list.
+contrast_val = [
+    np.array([[-1, -1, 1, 1]]),  # run 1
+    np.array([[-1, -1, 1, 1]]),  # run 2
+]
+
+z_map = fmri_glm_multises.compute_contrast(
+    contrast_val,
+    output_type="z_score",
+)
+plotting.plot_stat_map(
+    z_map,
+    bg_img=mean_img_,
+    threshold=3.0,
+    cut_coords=cut_coords,
+    title=f"{contrast_id}, fixed effects",
+)
+
+plotting.show()
+
+# %%
+# You may note that the results are the same as the first fixed effects
+# analysis, but with a lot less code.
+
+# %%
+# Compute a range of contrasts across both runs
+# ---------------------------------------------
+# It may be useful to investigate a number of contrasts.
+# Therefore, we will move beyond the original contrast of interest
+# and both define and compute several.
+
+# %%
+# Contrast specification
+n_columns = design_matrices[0].shape[1]
+contrasts = {
+    "SStSSp_minus_DStDSp": np.array([[1, 0, 0, -1]]),
+    "DStDSp_minus_SStSSp": np.array([[-1, 0, 0, 1]]),
+    "DSt_minus_SSt": np.array([[-1, -1, 1, 1]]),
+    "DSp_minus_SSp": np.array([[-1, 1, -1, 1]]),
+    "DSt_minus_SSt_for_DSp": np.array([[0, -1, 0, 1]]),
+    "DSp_minus_SSp_for_DSt": np.array([[0, 0, -1, 1]]),
+    "Deactivation": np.array([[-1, -1, -1, -1, -1]]),
```

Comment:
Changed this from the original, 

```python
"Deactivation": np.array([[-1, -1, -1, -1, 4]]),
```

because for me deactivations are just obtained with -1 and not by 'contrasting' conditions: am I wrong here?

Similarly, I'd be tempted to have the effect of interest be 

```python
np.eye(n_columns)[:4, :]
```

as the fifth conditions is of no interest in the experiment.